src/site/_includes/spaces.njk

```diff
@@ -0,0 +1,158 @@
+---
+layout: default-next
+---
+<div class="landing-page spaces">
+
+  <header>
+    <div class="hero wrapper">
+      <div class="hero__columns switcher">
+        <div>
+          <div class="flow">
+              <h1 class="text-size-6">{{ title }}</h1>
+              <p>{{ description }}</p>
+          </div>
+        </div>
+        <div class="flow">
+          {% Img src="image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/Rm7d6KWe3gabmU9IqZeq.png", alt="Globe", width="800", height="650" %}
+        </div>
+      </div>
+    </div>
+  </header>
+
+{% set spaces = collections['twitter-space'] %}
+<div class="spaces wrapper region all-center">
+  <h2 class="text-size-4  gap-bottom-size-2">{{ 'i18n.spaces.upcoming_spaces' | i18n(locale) }}</h2>
+  <div class="auto-grid all-center">
+
+  {% set upcomingSpaces = spaces | livePosts | filterInUpcoming('event_date') %}
+  {% set pastSpaces = spaces | livePosts | filterOutUpcoming('event_date') %}
+
+  {% if not upcomingSpaces.length %}
+  <p>A friendly message reminding to check for new upcomig spaces soon</p>
+  {% endif %}
+
+  {% for space in upcomingSpaces %}
+    {% if loop.index0 < 6 %}
+    <article class="card flow">
+      <div class="card__header">
+        <div class="avatars">
+        {% for host in space.data.hosts %}
+          <img class="avatar" src="{{ host.image }}" alt="alt" decoding="async" height="40" loading="lazy" width="40">
+        {% endfor %}
+        </div>
+      </div>
+      <div class="card__content flow">
+        <h2 class="card__heading text-size-3">
+          {{ space.data.title }}
+        </h2>
+        <ul class="hosts">
+          {% for host in space.data.hosts %}
+            <li>
+                {{ host.name }}
+              {% if host.host %} <span class="pill" data-inactive>{{ 'i18n.spaces.host' | i18n(locale) }}</span> {% endif %}
+            </li>
+          {% endfor %}
+        </ul>
+        <time class="eyebrow">{{ space.data.event_date | prettyDate('DATE_FULL') }}</time>
+        <p class="gap-top-size-1 gap-bottom-size-2">{{ space.data.description }}</p>
+      </div>
+      {% if space.data.twitter_link %}
+        <a class="button" href="{{ space.data.twitter_link }}" target="_blank">
+          {% include "icons/calendar.svg" %}
+          <span>{{ 'i18n.spaces.see_on_twitter' | i18n(locale) }}</span>
+        </a>
+      {% else %}
+        <a class="button" href="{{ space.data.event_date | calendarLink(space.data.title, space.data.description) }}" target="_blank">
```

Comment:
I know that support for alternative calendars can be tough to code, but it's not clear from the link that "Add to Calendar" will only work with Google Calendar.

I'd recommend putting something in the UI to clarify that this functionality will only work with Google Calendar.

Reply:
I'll ask designers to update the icon/button, but I would prefer not to block on this.